robots/breadboxBot.py

```diff
@@ -52,9 +55,11 @@ def __init__(self, period: float = 0.02) -> None:
         wpilib.SmartDashboard.setPersistent("Auto Distance 1")
         wpilib.SmartDashboard.setPersistent("Auto Distance 2")
 
+
     def getAutonomousCommand(self):
+        self.Position.getPosition()
         return(Autonomous(self.driveTrain, self.navx, self.robot_arm_controller, self.robot_Grabber))
```

Comment:
I THINK this should look like the following:
```suggestion
        return(Autonomous(self.driveTrain, self.navx, self.robot_arm_controller, self.robot_Grabber, self.Position.getPosition()))
```